src/core/StateSerializationHelper.cpp

```diff
@@ -114,15 +114,26 @@ bool StateSerializationHelper::load( const QString& filename )
                       filename.toStdString().c_str(), e.what( ));
             return false;
         }
+
         ifs.close();
     }
 
-    ContentWindowPtrs contentWindows = state.getContentWindows();
+    ContentWindowPtrs& contentWindows = state.getContentWindows();
     if( state.getVersion() < FIRST_PIXEL_COORDINATES_FILE_VERSION )
-        scaleToDisplayGroup( contentWindows );
-    validate( contentWindows );
+            scaleToDisplayGroup( contentWindows );
+        validate( contentWindows );
+
+
+    if( state.getVersion() <  WINDOW_TITLES_VERSION )
+    {
+        state.getDisplayGroup().setX( displayGroup_->x());
```

Comment:
implement an assignment operator or a function that accepts the display group and assigns the values in there? This exposure here looks wrong.